python/GafferSceneUI/LightEditor.py

```diff
@@ -415,6 +415,37 @@ def __toggleBoolean( self, inspectors, inspections ) :
 
 		return True
 
+	def __activeInspectionTweaks( self, pathListing ) :
+
+		tweaks = []
+
+		with Gaffer.Context( self.getContext() ) as context :
+			for columnSelection, column in zip( pathListing.getSelection(), pathListing.getColumns() ) :
+				if not isinstance( column, _GafferSceneUI._LightEditorInspectorColumn ) :
+					continue
+				for path in columnSelection.paths() :
+					context["scene:path"] = GafferScene.ScenePlug.stringToPath( path )
+					inspection = column.inspector().inspect()
+					if (
+						inspection is not None and
+						inspection.editable() and
+						inspection.sourceType() == GafferSceneUI.Private.Inspector.Result.SourceType.EditScope
+					) :
+						edit = inspection.acquireEdit()
+						if isinstance( edit, Gaffer.TweakPlug ) and edit["enabled"].getValue() :
+							tweaks.append( edit )
+
+		return tweaks
+
+	def __removeEdits( self, pathListing ) :
+
+		with Gaffer.UndoScope( self.__settingsNode["in"].getInput().ancestor( Gaffer.ScriptNode ) ) :
```

Comment:
We have simpler access to the ScriptNode via `self.scriptNode()`.

Reply:
Fixed in https://github.com/GafferHQ/gaffer/pull/4948/commits/3494800095112a1673280dcfe1149485b5a5177c